src/main/java/graphql/Directives.java

```diff
@@ -54,11 +63,39 @@ public class Directives {
                         newInputValueDefinition()
                                 .name("reason")
                                 .description(createDescription("The reason for the deprecation"))
-                                .type(newTypeName().name("String").build())
+                                .type(newTypeName().name(STRING).build())
                                 .defaultValue(StringValue.newStringValue().value(NO_LONGER_SUPPORTED).build())
                                 .build())
                 .build();
 
+        INCLUDE_DIRECTIVE_DEFINITION = DirectiveDefinition.newDirectiveDefinition()
+                .name(INCLUDE)
+                .directiveLocation(newDirectiveLocation().name(FRAGMENT_SPREAD.name()).build())
+                .directiveLocation(newDirectiveLocation().name(INLINE_FRAGMENT.name()).build())
+                .directiveLocation(newDirectiveLocation().name(FIELD.name()).build())
+                .description(createDescription("Directs the executor to include this field or fragment only when the `if` argument is true"))
+                .inputValueDefinition(
+                        newInputValueDefinition()
+                                .name("if")
+                                .description(createDescription("Included when true."))
+                                .type(newNonNullType(newTypeName().name(BOOLEAN).build()).build())
+                                .build())
```

Comment:
A comment for myself: in AST land, DirectiveDefinitions ONLY have "inputValueDefinition", this is the AST type used to model "arguments". Confusingly there is another `graphql.language`/AST class called "Argument". It is meant to be `inputValueDefinition` here.